packages/studio/src/components/UndefinedMenuButton.tsx

```diff
@@ -70,9 +70,9 @@ export default function UndefinedMenuButton({
           aria-label="Toggle undefined value menu"
         />
         {isOpen && (
-          <div className="absolute mt-3">
+          <div className="relative mt-3 left-[134px]">
             <ul
-              className="relative bg-white rounded border z-10 shadow-2xl text-sm"
+              className="absolute bg-white rounded border z-10 shadow-2xl text-sm"
```

Comment:
small thing, but I think with these changes, the dropdown gets cut off if it extends past the last prop editor
![image](https://github.com/yext/studio/assets/59857107/4757f7c9-4904-465b-92f2-e1349a9ce122)